salt/utils/verify.py

```diff
@@ -194,13 +194,13 @@ def verify_files(files, user):
     return True
 
 
-def verify_env(dirs, user, permissive=False, pki_dir='', skip_extra=False):
+def verify_env(dirs, user, permissive=False, sensitive_dirs=None, skip_extra=False):
```

Comment:
@kstreee For the future, whenever we change the API like this, the previous option needs to be put on a deprecation path and both options should be handled. This will break calls to `verify_env` that don't have this change in place for other users.

I'll add a fix right now, but I just wanted to point this out for future changes. :)

Reply:
See #44584